Sources/SearchResultsV2.swift

```diff
@@ -0,0 +1,195 @@
+//
+//  SearchResultsswift
+//  InstantSearchCore-iOS
+//
+//  Created by Vladislav Fitc on 28/02/2019.
+//  Copyright © 2019 Algolia. All rights reserved.
+//
+
+import Foundation
+
+
+// Temporary enum without cases for defining namespace
+enum V2 {
+    
+    public struct SearchResults<T: Decodable>: Decodable {
+        
+        enum CodingKeys: String, CodingKey {
+            case totalHitsCount = "nbHits"
+            case page
+            case facets
+            case pagesCount = "nbPages"
+            case hits
+            case hitsPerPage
+            case processingTimeMS
+            case query
+            case queryID
+            case areFacetsCountExhaustive = "exhaustiveFacetsCount"
+            case message
+            case queryAfterRemoval = "queryAfterRemoval"
+            case aroundGeoLocation = "aroundLatLng"
+            case automaticRadius
+            case facetStats = "facets_stats"
+        }
+        
+        /// Hits.
```

Comment:
You can specify in the comment that it's the hits for the specific page specified.